bin/unit_test_go

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+
+ginkgo src/drivers
```

Comment:
I'd rather not enumerate all nested directories that contain unit tests here. Can we use something like `ginkgo src/...` or `ginkgo src/*` or `ginkgo src` here to run all unit tests in `src`?